### ISSUE TITLE
[COGNITION]: Direct Deposit - Fix an anchor link to outside content page

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#1335]
## Description

One of the links in the learn more accordions has an anchor target on the URL ( https://staging.va.gov/change-direct-deposit/#mail-phone ) The page this link references should have a heading or a div with an `id="mail-phone"` so the link resolves to the right page, and the right spot on the page. Screenshot attached.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* As a user, I want the link to point to the right section on the content page when I click it

## WCAG or Vendor Guidance (optional)

* [Link Purpose (In Context): Understanding SC 2.4.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html)

## Screenshots or Trace Logs

![Screen Shot 2019-08-27 at 4.19.59 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/5072cdae-2cda-4808-8660-9ef306929de9)

**Comment by erikphansen** [comment]
@Justin-Pickett feel free to validate this and close it.

**Comment by Justin-Pickett** [comment]
This has been validated and works as intended. Closing this ticket.
